tests/TestCase/Command/BakeMigrationDiffCommandTest.php

```diff
@@ -24,7 +26,22 @@
 use Migrations\Migrations;
 use Migrations\Test\TestCase\Shell\TestCompletionStringOutput;
 use Migrations\Test\TestCase\TestCase;
+use Migrations\Command\BakeMigrationDiffCommand;
 
+class CustomBakeMigrationDiffCommand extends BakeMigrationDiffCommand
+{
+    public $pathFragment = 'config/MigrationsDiff/';
```

Comment:
I extended `BakeMigrationDiffCommand`  to set `pathFragment` and hacking  `getDumpSchema`